docs/handbook/getting-started.md

```diff
@@ -1417,7 +1417,45 @@ Use the following steps to change the root password:
     These pages are only available by running the man page viewer locally on your system.
 </div>
 
-### Introduction
+### Quick Start to IPS
+
+As a dekstop user, you will likely want to enable the `Software Installation` profile for
+your user; you can use `sudo` instead of `pfexec` if you are just getting started with
+OpenIndiana and have not yet familiar with roles
+(replace `username` with your actual username):
```

Comment:
Software Installation is a profile (pfexec = profiled execute, other profiled commands are available, too) while roles are a different thing (if you have been granted a role you can su into it; so roles are special user accounts that cannot login directly). Privileges are the third concept which are low level rights ("fine grain control over the actions of processes" - see man privileges).

Reply:
Got it - I was using `roles` as a broad term but I likely should have said "illumos RBAC". Is that an acceptable change, or would you like me to integrate your comment into the same paragraph?